cmd/ddev/cmd/auth-ssh.go

```diff
@@ -173,6 +177,28 @@ func fileIsPrivateKey(filePath string) bool {
 	return string(buffer) == string(prefix)
 }
 
+// getCertificateForPrivateKey returns path and name for optional OpenSSH certificate
+// https://www.man7.org/linux/man-pages/man1/ssh-keygen.1.html#CERTIFICATES
+// https://github.com/ddev/ddev/issues/6832
+func getCertificateForPrivateKey(path string, name string) (string, string) {
+	cert := path + "-cert.pub"
+	if !fileutil.FileExists(cert) {
+		return "", ""
+	}
+	certPath, err := filepath.EvalSymlinks(cert)
+	if err != nil {
+		util.Warning("Unable to read %s file: %v", cert, err)
+		return "", ""
+	}
+	if !fileutil.FileIsReadable(certPath) {
+		util.Warning("Unable to read %s file: file is not readable", certPath)
+		return "", ""
+	}
+	certPath = util.WindowsPathToCygwinPath(certPath)
```

Comment:
Wow, can't believe you got this in there too. Does `ddev auth ssh` even work on git-bash?